packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -157,6 +157,6 @@ export class AtomicResultList implements InitializableComponent {
       return this.renderPlaceholder();
     }
 
-    return this.results;
+    return [this.templateHasError && <slot></slot>, this.results];
```

Comment:
Why is a slot added on error?

Reply:
the slots are the result templates, which display their respective errors